<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>855</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administrator Settings</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="tab_account">
   <attribute name="title">
    <string>Account Management</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="op_label">
            <property name="text">
             <string>Operator</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="op_list">
            <item>
             <property name="text">
              <string>Jack</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kate</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="level_label">
            <property name="text">
             <string>User Level</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="level_list">
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Administrator</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,8">
        <item>
         <widget class="QPushButton" name="btn_add">
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_delete">
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fill_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_led">
   <attribute name="title">
    <string>LED Settings</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,5,5,5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_fill">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,6,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="bright_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Brightness&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_left_slider">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="bright_slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_right_slider">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;100%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,4,1">
        <item>
         <widget class="QLabel" name="label_ondelay">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;On Delay&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_ondelay"/>
        </item>
        <item>
         <widget class="QLabel" name="units1">
          <property name="text">
           <string>uSec</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,4,1">
        <item>
         <widget class="QLabel" name="label_offdelay">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Off Delay&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_offdelay"/>
        </item>
        <item>
         <widget class="QLabel" name="units2">
          <property name="text">
           <string>uSec</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_camera">
   <attribute name="title">
    <string>Camera Settings</string>
   </attribute>
   <widget class="QPushButton" name="btn_cfgcam">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Camera Settings from Directory</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cfgcam</sender>
   <signal>clicked()</signal>
   <receiver>TabWidget</receiver>
   <slot>configCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>727</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>configCamera()</slot>
 </slots>
</ui>
